.github/workflows/datafuse-perf-docker.yml

```diff
@@ -13,6 +13,25 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Rust setup
+        run: |
+          bash ./scripts/setup/dev_setup.sh
+
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
```

Comment:
This may not a good idea for Rust 2021

Reply:
we could bypass all caches by running build on self hosted runner